astropy/units/tests/test_format.py

```diff
@@ -429,9 +489,14 @@ def test_vounit_custom():
 
 
 def test_vounit_implicit_custom():
-    x = u.Unit("furlong/week", format="vounit")
+    # Yikes, really...
```

Comment:
Why the "yikes"? Can you please clarify?

Reply:
Clarified:
```
    # Yikes, this becomes "femto-urlong"...  But at least there's a warning.
```